types/react-redux/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for react-redux 4.4.0
+// Type definitions for react-redux 4.4.47
 // Project: https://github.com/rackt/react-redux
 // Definitions by: Qubo <https://github.com/tkqubo>, Sean Kelley <https://github.com/seansfkelley>, Thomas Hasner <https://github.com/thasner>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.4.1
```

Comment:
you might wanna change this in a separate PR so you don't have to update whatever else is breaking the build